core/src/test/scala/stryker4s/run/threshold/ThresholdCheckerTest.scala

```diff
@@ -0,0 +1,122 @@
+package stryker4s.run.threshold
+
+import stryker4s.Stryker4sSuite
+import stryker4s.config.{Config, Thresholds}
+import stryker4s.extensions.exceptions.InvalidThresholdValueException
+import stryker4s.model.MutantRunResults
+import stryker4s.scalatest.LogMatchers
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+class ThresholdCheckerTest extends Stryker4sSuite with LogMatchers {
+
+  describe("thresholds") {
+    def interceptThresholdException(high: Int = 80, low: Int = 60, break: Int = 0): Unit = {
+      intercept[InvalidThresholdValueException] {
```

Comment:
[Using exception matchers is more in line with the rest of the tests](http://www.scalatest.org/user_guide/using_matchers#expectedExceptions)